seirmo/figures.py

```diff
@@ -0,0 +1,121 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import plotly.graph_objects as go
+
+import warnings
+
+
+class IncidenceNumberPlot(object):
+    """
+
+    A figure class that illustrates the incidence number time
+    series as a bar plot over time.
+
+    """
+    def __init__(self):
+        super(IncidenceNumberPlot, self).__init__()
+
+        # Instantiate an empty plotly figure
+        self._fig = go.Figure()
+
+    def add_data(self, data, time_key='Time',
+                 inc_key='Incidence Number'):
+        """
+
+        Add a bar plot trace for the given incidence number data
+        in the figure
+
+        Parameters
+        ----------
+        data: A pandas.DataFrame with two columns, one being time points,
```

Comment:
I think the way sphinx works is that you write
```
Parameters
-----------
agrument 1
    Description argument 1
argument 2
    Description argument 2
```

You can always check whether the docs render as expected by going into the docs directly in the terminal and then type 
```
make clean; make html
```
This builds the docs locally and you can look at the index.html in the build directory whether the docs have rednered properly.